packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/fragments/Request.kt

```diff
@@ -27,6 +27,7 @@ data class Request(
     var sort: Array<Sort> = arrayOf(),
     var having: Array<Having> = arrayOf(),
 
+    var dataSource: String = "",
```

Comment:
Should this have a value? Seems like as is it would make navi use `''` as `dataSource` instead of letting it fall back to default

Reply:
You're right, will fix